src/viewer.js

```diff
@@ -3415,6 +3417,43 @@ function onCanvasPinch( event ) {
     }
 }
 
+function onCanvasFocus( event ) {
+
+    /**
+     * Raised when a pointer focus on the {@link OpenSeadragon.Viewer#canvas} element.
```

Comment:
I think a better phrasing would be `Raised when the {@link OpenSeadragon.Viewer#canvas} element gets keyboard focus.`